mycore-validation/src/main/java/org/mycore/validation/pdfa/MCRPDFAFunctions.java

```diff
@@ -82,7 +81,8 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
                     try {
                         results.put(dir.relativize(file).toString(), PDF_A_VALIDATOR.validate(file));
                     } catch (MCRPDFAValidationException e) {
-                        throw new MCRException(e);
+                        results.put(dir.relativize(file).toString(), new ValidationErrorResult());
+
```

Comment:
Details (cause, stacktrace) of Exception are lost here. Better way is to put the Exception into the map and serialize at least the cause into the Document. You could use a record to old ValidationResult or Exception and use it as a map value.